documentation/configuringDynaflowLauncher/DynaflowLaunchersInputFiles.tex

```diff
@@ -131,14 +133,36 @@ \subsection{Configuration file}
 \small{StopTime} & \small{integer} & \small{Simulation stop time (in s)} & \small{100} \\
 \rowcolor{gray!10}
 \small{TimeStep} & \small{real} & \small{Maximum time solver step value} & \small{10} \\
+\rowcolor{white}
+\multirow{3}{*}{\small{ChosenOutputs}} & \multirow{3}{*}{\small{struct}} & \multirow{2}{*}{\small{Chosen outputs to configure}} & \small{Depends on the} \\
+& & \multirow{2}{*}{\small{the .jobs file}} & \small{buildtype and the} \\
+& & & \small{simulation kind} \\
 \bottomrule
 \end{tabular}
 \caption{Simulation parameters}
 \end{table}
 
+\definecolor{darkblue}{rgb}{,0,0,1}
+\begin{table}[H]
+The default chosen outputs depend on the build type (Debug or Release) and the simulation kind (Steady State or Security Analysis). If we want, the \textit{ChosenOutput} attribute array in the configuration file can add optional chosen outputs.
+\center
+\begin{tabular}{| c | c | c |}
+\cline{2-3}
+\multicolumn{1}{c|}{} & Steady State & Security Analysis \\
+\hline
+\multirow{2}{*}{Debug} & \textcolor{darkblue}{\small{IIDM CONSTRAINTS}} & \textcolor{darkblue}{\small{IIDM CONSTRAINTS}} \\
```

Comment:
Please put Release first as it is the most common usage. Also, please explain with an example the syntax to add the options in the config file. 
I would also prefer to change the colors: green for default and grey for optionals

Reply:
I added chosenOutputs to resources/syntaxExample/config.json to show the correct syntax.